# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/compositing/mix-blend-mode/mix-blend-mode-both-parent-and-blended-with-3D-transform.html.ini
# Commit: eefb300b7640
# Full Hash: eefb300b7640986be7d1a4a23c133c4ebe73261d
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2019-05-06 21:46:17
# Regressor Bug: 1540200
# File Overlap Count: 1
# Description:
#   Bug 1540200 - Part 2. Calculate snapped primitive rect and offsets for WebRender on the CPU. r=kvark
#   
#   We currently do most snapping on the GPU in the shader. However the
#   picture's local rect needs to take into account the snapping done there,
#   so we need to calculate this earlier in the pipeline. Instead of using
# ==============================================================================

diff -r 403ea69a03b0 -r eefb300b7640 testing/web-platform/meta/css/compositing/mix-blend-mode/mix-blend-mode-both-parent-and-blended-with-3D-transform.html.ini
--- a/testing/web-platform/meta/css/compositing/mix-blend-mode/mix-blend-mode-both-parent-and-blended-with-3D-transform.html.ini	Wed Apr 10 12:35:20 2019 -0400
+++ b/testing/web-platform/meta/css/compositing/mix-blend-mode/mix-blend-mode-both-parent-and-blended-with-3D-transform.html.ini	Wed Apr 10 14:20:34 2019 -0400
@@ -1,3 +1,3 @@
 [mix-blend-mode-both-parent-and-blended-with-3D-transform.html]
   expected:
-    if os == "android" and not e10s: FAIL
+    if webrender or (os == "android" and not e10s): FAIL
